content/faq.md

```diff
@@ -1,9 +1,29 @@
-title: complate Frequently Asked Questions 
+title: complate Frequently Asked Questions
 h1: FAQ
-subtitle: Frequently asked questions
+subtitle: Frequently Asked Questions
 safe: false
 
 
+Is complate Right for Me? <span id="trade-offs"></span>
+-------------------------
+
+While complate has many advantages, it also introduces additional complexity:
+The use of JavaScript typically requires both a compiler (for module resolution and to transform JSX) and
+a JS runtime environment (typically provided via [the respective adaptor](getting-started.html)) --
```

Comment:
That's a forward reference, as Getting Started doesn't provide such guidance yet.